src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseFilesService.cs

```diff
@@ -703,7 +707,7 @@ private async Task<DataFileInfo> GetDataFileInfo(Guid releaseId, ReleaseFileRefe
 
             // If the file does exist then it could possibly be
             // partially uploaded so make sure meta data exists for it
-            if (blob.GetUserName().IsNullOrEmpty())
+            if (string.IsNullOrEmpty(blob.GetUserName()))
```

Comment:
Perhaps we should just make this our own extension function?

Reply:
Cool, I've added it.